packages/celotool/genesis_rc0.json

```diff
@@ -0,0 +1,297 @@
+  "difficulty": "0x1",
+  "coinbase": "0x0000000000000000000000000000000000000000",
+  "alloc": {
+    "04acedA1F7909187796D6d59F842ED4aEEa3E4b6": {
```

Comment:
There are a bunch of balances here that shouldn't be there, these look like the ones that celotool adds automatically for testnets

Reply:
These look like the 70 validators generated from the mnemonic and their balances are defined here 
https://github.com/celo-org/celo-monorepo/blob/0ea99f3a94dea0eb9bf9dd89c300cb4390006ee3/packages/celotool/src/lib/generate_utils.ts#L109

Should I change val0 to be the same as the rest?

Reply:
None of them should have any balance